src/main/java/org/folio/consortia/repository/UserTenantRepository.java

```diff
@@ -24,8 +24,18 @@ public interface UserTenantRepository extends JpaRepository<UserTenantEntity, UU
 
   boolean existsByTenantId(String tenantId);
 
-  @Query("SELECT ut FROM UserTenantEntity ut WHERE ut.userId= ?1 AND ut.isPrimary= ?2")
-  Optional<UserTenantEntity> findByUserIdAndIsPrimary(UUID userId, Boolean isPrimary);
+  @Query("SELECT ut FROM UserTenantEntity ut WHERE ut.userId= ?1 AND ut.isPrimary= true")
+  Optional<UserTenantEntity> findByUserIdAndIsPrimaryTrue(UUID userId);
+
+  @Query("SELECT ut FROM UserTenantEntity ut WHERE ut.userId NOT IN (SELECT ut.userId FROM UserTenantEntity ut WHERE ut.userId= ?1 AND ut.isPrimary=true) AND ut.userId= ?1")
+  List<UserTenantEntity> getOrphanUserTenantAssociationsByUserIdAndIsPrimaryFalse(UUID userId);
+
+  @Query("SELECT ut FROM UserTenantEntity ut WHERE ut.userId= ?1 AND ut.isPrimary= false")
+  List<UserTenantEntity> getByUserIdAndIsPrimaryFalse(UUID userId);
```

Comment:
After becoming these method names similar, changes its name and logic in order to make them implying their function with their name

Reply:
Old style is more readable as for me, very long method names hard to read